src/partr.c

```diff
@@ -299,18 +131,6 @@ void jl_threadfun(void *arg)
 }
 
 
-// enqueue the specified task for execution
-JL_DLLEXPORT int jl_enqueue_task(jl_task_t *task)
-{
-    char failed;
-    if (multiq_insert(task, task->prio) == -1)
-        failed = 1;
-    failed = 0;
-    JL_PROBE_RT_TASKQ_INSERT(jl_current_task->ptls, task);
```

Comment:
Losing this is disappointing :) We will need to find a way to implement this from Julia efficiently.

Reply:
Could we implement the logic of https://github.com/JuliaPerf/UProbes.jl directly in codegen?

Reply:
We could, but bpftrace would still not find the new probes. UProbes.jl works by outlining the probe to a shared library that bpftrace and co. can scan.